FiftyOne.Common.CloudStorage/FiftyOne.Common.CloudStorage.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Include="..\README.md">
+    <None Include="..\README-CloudStorage.md">
```

Comment:
let's keep the name README.md but move inside FiftyOne.CommonCloudStorage folder